app/assets/javascripts/components/markdown-toolbar.js

```diff
@@ -620,7 +620,7 @@
     var text = textarea.value.slice(textarea.selectionStart, textarea.selectionEnd);
 
     var result = void 0;
-    if (styleArgs.orderedList) {
+    if (styleArgs.orderedList && text) {
```

Comment:
There's a slight difference in behaviour between this one and bullet points: clicking the button multiple times for bullets (or quotes) inserts on a new line; clicking for a numbered list just keeps adding to the same line.

Reply:
Updated the code so they behave the same (adding a new line after the item - as it's needed so the markdown is rendered correctly)